quadtree/quadtree.go

```diff
@@ -262,18 +264,24 @@ func (q *Quadtree) FindMatching(p *geo.Point, f Filter) geo.Pointer {
 // FindKNearestMatching returns k closest Value/Pointer in the quadtree for which
 // the given filter function returns true. This function is thread safe.
 // Multiple goroutines can read from a pre-created tree.
-func (q *Quadtree) FindKNearestMatching(p *geo.Point, k int, f Filter) []geo.Pointer {
+//
+// FindKNearestMatching allows defining a maximum distance in order to reduce search iterations.
```

Comment:
nit: double comment, I don't think this will display the first part in godoc, right?